front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -39,11 +39,10 @@ export default function SpeedLimitByTagSelector({
       >
         <img width="32px" src={icon} alt="speedometer" />
         <span className="text-muted">{t('speedLimitByTag')}</span>
-        <SelectImprovedSNCF
-          sm
+        <Select
+          small
           disabled={disabled}
-          withSearch
-          dataTestId="speed-limit-by-tag-selector"
+          id="speed-limit-by-tag-selector"
           value={speedLimitByTag || t('noSpeedLimitByTag').toString()}
```

Comment:
Would be nicer to not use strings intended for humans (such as translations) as `<select>` values (which are meant to be unique and are not displayed to the user). A simple solution is to move `|| t('noSpeedLimitByTag').toString()` to `getOptionLabel()`.

Reply:
@emersion can you check the fix and if it's done properly

Reply:
Yeah, sounds good!